app/src/main/res/values/strings.xml

```diff
@@ -172,4 +172,6 @@
     <string name="cd_navdrawer_user_picture">User Profile Image</string>
     <string name="title_signing_needed">Sign-in Needed</string>
     <string name="signin">Sign-in</string>
+    <string name="feedback_message_required">Message required</string>
+    <string name="feedback_invalid_email">Input valid email address</string>

```

Comment:
It is a tough problem. It already happened for the Lead messages wording changes. 
The translators on crowdin sees those changed strings untranslated with a quick suggestion with the old translation. 

What we can do is that we can wait for the new translations. Or just use the old ones.